test/e2e/tests/collector_pipeline/main_test.go

```diff
@@ -23,7 +23,7 @@ func TestMain(m *testing.M) {
 	// Create KinD Cluster
 	util.Testenv.Setup(
 		envfuncs.CreateKindClusterWithConfig(util.KindClusterName, util.NodeVersion, "../../kind-config.yaml"),
-		envfuncs.CreateNamespace(util.TestNamespace),
+		envfuncs.CreateNamespace(util.EraserNamespace),
```

Comment:
we don't need to create the namespace before in the tests besides for metrics and exclusion list so we should be able to delete this in the rest

Reply:
This is required now, because it sets `cfg.Namespace()` in the body of the e2e tests, so we need to create one namespace per e2e test